components/match2/commons/get_match_group_copy_paste.lua

```diff
@@ -173,4 +173,21 @@ function copyPaste.matchlist(frame, args)
 	return '<pre class="selectall" width=50%>' .. mw.text.nowiki(out) .. '</pre>'
 end
 
+function copyPaste.singleMatch(frame, args)
+	if not args then
+		args = getArgs(frame)
+	end
+
+	local bestof = tonumber(args.bestof or 3) or 3
+	local opponents = tonumber(args.opponents or 2) or 2
+	local mode = WikiSpecific.getMode(args.mode)
+
+	local out = WikiSpecific.getStart(nil, copyPaste.generateID(), 'singlematch', args)
+
+	out = out .. '\n|' ..
```

Comment:
`|M1=`

Reply:
i adjusted the template some days ago to allow input as `|M1=`or `|1=` (and hence `|`)

question is what should we set as standard for the copy paste
`|M1=` might indicate to some people that they can add more matches (which is not the case)

Reply:
{{1|}} is fine